### ISSUE TITLE
Recent Medium posts are not showing on the website

### ISSUE THREAD

**Issue by yurishkuro** [jaegertracing/documentation#246]
For some reason the latest Medium posts don't show on Jaeger website, i.e. at least two more are published since Tuning Jaeger Performance. 

fyi @pavolloffay @jpkrohling @lucperkins

**Comment by lucperkins** [comment]
There’s no way to debug or test the webhook-based system I put in place, and I’m not sure why it doesn’t work. I think that manual re-deploys are the only way to go here.

**Comment by yurishkuro** [comment]
I did a re-deploy, it didn't help.

**Comment by lucperkins** [comment]
@yurishkuro The most recent posts are up. Make sure to clear the cache.

**Comment by yurishkuro** [comment]
Hold on. Do we build the list of posts dynamically on page load or during Hugo build? If the latter, what kind of caching policy do we use for rendered HTML pages?

**Comment by yurishkuro** [comment]
Same issue again, the [Ticketmaster article][1] went up yesterday, I did a build yesterday, and just did another re-build, but the post still doesn't appear on the homepage. Also, the build suspiciously logged this at the end:
```
11:07:25 AM: 0 new files to upload
11:07:25 AM: 0 new functions to upload
11:08:05 AM: Finished processing build request in 1m21.485407702s
11:08:05 AM: Shutting down logging, 0 messages pending
```

[1]: https://medium.com/jaegertracing/ticketmaster-traces-100-million-transactions-per-day-with-jaeger-38ec6cf599f0

**Comment by jpkrohling** [comment]
I got the same with my latest blog post and I got around by clearing Netlify's build cache.

**Comment by yurishkuro** [comment]
I  did that yesterday, didn't help, but it was soon after publishing. Let me try again.

**Comment by yurishkuro** [comment]
ok that helped.
